### ISSUE TITLE
Show graph is not always available in the menu

### ISSUE THREAD

**Issue by dennisschoenmakers** [quintel/etmodel#2088]
When you open a new scenario, the menu item 'show graph' is missing. You only have this when you created a new scenario, not when you loaded an existing scenario, or when you hit refresh after loading a new scenario.

The problem seems to be in the `api_session_id` [not being present] when the page is first loaded on a new scenario:
(https://github.com/quintel/etmodel/blob/master/app/views/layouts/etm/_settings_menu.html.haml#L31-L33):

``` Ruby
- if Current.setting.api_session_id.present?
  %li.sep
    = link_to t("header.show_graph"), et_engine_graph_url, target: "_blank"
```

spotted by @AlexanderWirtz, cc: @jorisberkhout, @ChaelKruip

**Comment by ChaelKruip** [comment]
Assigning @antw as well.

**Comment by antw** [comment]
Was fixed by c412c87.
